### PR TITLE
Updated Readme and deleted out of date content

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated Readme and deleted out of date content

Unsure if the `docs/code-coverage.md` doc is still relevant - can a reviewer let me know if this should be retained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,48 +4,34 @@
...
-Ubuntu: 
-
-1) Download Gradle
-```shell script
-wget https://services.gradle.org/distributions/gradle-5.0-bin.zip -P /tmp
-sudo unzip -d /opt/gradle /tmp/gradle-*.zip
-```
-2) Setup environment variables
+* [Contribution Guidelines](CONTRIBUTING.md)
+* [Coding Conventions](https://github.com/hyperledger/besu/blob/master/CODING-CONVENTIONS.md)
```
Comment (ajsutton) [inline README.md:27]: This link is a 404.

File: README.md [hunk 2]
```diff
@@ -4,48 +4,34 @@
...
 
-```shell script
-sudo chmod +x /etc/profile.d/gradle.sh
-source /etc/profile.d/gradle.sh
-```
+### Install Prerequisites
 
-OSX: `brew install gradle`
+* Java 11
+* Gradle
```
Comment (ajsutton) [inline README.md]: Gradle is not required - the `gradlew` script will automatically retrieve it.

File: README.md [hunk 3]
```diff
@@ -77,85 +63,20 @@ git clone https://github.com/PegaSysEng/teku.git
...
-Xinterop-owned-validator-count: 8
-Xinterop-start-state: "/tmp/genesis.ssz"
-Xinterop-number-of-validators: 16
-Xinterop-enabled: True
-```
-
-## Deposit Simulation
-
-Click [here](pow/README.md) for setup instructions
+Refer to `config/config.yaml` for a set of default configuration settings.
```
Comment (ajsutton) [inline README.md]: This is no longer correct.  `--help` is now the best way to see the default config settings.  `config/config.yaml` is only used by `run.sh` I think.

File: docs/code-coverage.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ## Code coverage
 
-We use the jacoco test coverage plugin, which will generate coverage data whenever tests are run.
+We use the jacoco test coverage plugin, which generates coverage data whenever tests are run.
 
 To run the report:
 ```
```
Comment (ajsutton) [inline docs/code-coverage.md]: I'd delete this.  It probably works but we never use it so can't be sure.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (MadelineMurray) [reply README.md:27]: Fixed

File: README.md (hunk 2 above)
Comment (MadelineMurray) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (MadelineMurray) [reply README.md]: Fixed

File: docs/code-coverage.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/code-coverage.md]: Deleted

**Pull request merged**